src/get-started/wordcount-example.md

```diff
@@ -773,12 +773,21 @@ To view the full code in Java, see
 
 > **Note:** WindowedWordCount is not yet available for the Python SDK.
 
-### Unbounded and bounded pipeline input modes
+### Unbounded and bounded data sets
 
 Beam allows you to create a single pipeline that can handle both bounded and
 unbounded types of input. If your input has a fixed number of elements, it's
```

Comment:
`bounded and unbounded input` or `bounded and unbounded input data sets`, for consistency